valhalla/baldr/traffictile.h

```diff
@@ -130,15 +132,15 @@ struct TrafficSpeed {
                          const uint32_t c2,
                          const uint32_t c3,
                          const bool incidents)
-      : overall_speed{overall_speed}, speed1{s1}, speed2{s2}, speed3{s3}, breakpoint1{b1},
-        breakpoint2{b2}, congestion1{c1}, congestion2{c2}, congestion3{c3},
-        has_incidents{incidents}, spare{0} {
+      : overall_encoded_speed{overall_encoded_speed}, encoded_speed1{s1}, encoded_speed2{s2},
+        encoded_speed3{s3}, breakpoint1{b1}, breakpoint2{b2}, congestion1{c1}, congestion2{c2},
+        congestion3{c3}, has_incidents{incidents}, spare{0} {
   }
 
   json::MapPtr json() const volatile {
     auto live_speed = json::map({});
     if (speed_valid()) {
-      live_speed->emplace("overall_speed", static_cast<uint64_t>(get_overall_speed()));
+      live_speed->emplace("overall_encoded_speed", static_cast<uint64_t>(get_overall_speed()));
```

Comment:
This one isn't encoded, it's accessed via the getter

Reply:
its also a breaking change to the locate API so we probably shouldnt change it